code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -135,15 +135,14 @@
 
 	var/list/turf/synd_spawn = list()
 
-	for(var/obj/effect/landmark/A in landmarks_list)
-		if(A.name == "Syndicate-Spawn")
-			synd_spawn += get_turf(A)
-			qdel(A)
-			A = null
-			continue
-
-	var/obj/effect/landmark/uplinklocker = locate("landmark*Syndicate-Uplink")	//i will be rewriting this shortly
-	var/obj/effect/landmark/nuke_spawn = locate("landmark*Nuclear-Bomb")
+	for(var/obj/effect/landmark/nukeops/syndicate_spawn/A in landmarks_list)
```

Comment:
This code's purpose is to delete either landmarks, I feel like it'd be a good idea to kill this and rework the `synd_spawn` stuff while you're at it.